### PR TITLE
Added default shoutout. Added error checking.

### PR DESCRIPTION
**Pull request by JamesTheHacker**
#7  @ryanwhittaker added some error checking, added default shoutout. Apologies for not commiting the changes separately.

### CODE REVIEW

**Comment by JamesTheHacker** [comment]
#9

File: fblive.php [hunk 1]
```diff
@@ -9,13 +9,25 @@
 $shoutouts = include __DIR__ . '/shoutouts.php';
 $settings = include __DIR__ . '/settings.php';
 
+/*
+ * Ensure the correct settings are set in order to run the script
+ */
+if(!isset($settings['ACCESS_TOKEN']) || empty($settings['ACCESS_TOKEN'])) {
```
Comment (ryanwhittaker) [inline fblive.php]:
```php
if (empty($settings['ACCESS_TOKEN'])) {
```

actually covers both of these cases in PHP. `empty` turns off error reporting, which means it also checks if the variable is set.

**Review by ryanwhittaker — APPROVED** [review]
Looks good to me and tested fine, just a minor comment. Merge it down when you wake up

**Comment by JamesTheHacker** [comment]
@ryanwhittaker check that. I've tested my end, works fine.

I've removed the `calculateXPos` function. The user can set the x and y coordinates manually as they may want to change the style of image completely. This also allows the user to create multiple `settings.php` files for different image styles. Might be worth passing in the settings file on the terminal.

Forgive me for not strictly following PSR2. I'm running out of time this end. I also should have created a new branch, it was too late to fix when I realised I hadn't.

**Pull request merged**